cmd/notary/tuf.go

```diff
@@ -568,14 +568,37 @@ func (ps passwordStore) Basic(u *url.URL) (string, string) {
 	return username, password
 }
 
-// getTransport returns an http.RoundTripper to be used for all http requests.
+// getTransport returns a non-admin http.RoundTripper to be used for all http requests.
 // It correctly handles the auth challenge/credentials required to interact
 // with a notary server over both HTTP Basic Auth and the JWT auth implemented
 // in the notary-server
 // The readOnly flag indicates if the operation should be performed as an
 // anonymous read only operation. If the command entered requires write
 // permissions on the server, readOnly must be false
+// This RoundTripper cannot perform admin requests, such as deleting repos
 func getTransport(config *viper.Viper, gun string, readOnly bool) (http.RoundTripper, error) {
+	trustServerURL := getRemoteTrustServer(config)
+	base, err := getBaseTransport(config)
+	if err != nil {
+		return nil, err
+	}
+	return tokenAuth(trustServerURL, base, gun, readOnly, false)
+}
+
+// getAdminTransport returns an admin http.RoundTripper to be used for all http requests,
+// and can be used for deleting repos
+func getAdminTransport(config *viper.Viper, gun string) (http.RoundTripper, error) {
```

Comment:
oh, sorry, saw the last arg for `tokenAuth` inverted too. Maybe we just update `getTransport` to take another arg?